### PR TITLE
Add linkify-it-py

### PR DESCRIPTION
**Pull request by chrisjsewell**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by chrisjsewell** [comment]
depends on https://github.com/conda-forge/staged-recipes/pull/13481

File: recipes/linkify-it-py/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% set name = "linkify-it-py" %}
+{% set version = "1.0.0" %}
```
Comment (chrisjsewell) [inline recipes/linkify-it-py/meta.yaml:2]:
```suggestion
{% set version = "1.0.1" %}
```

File: recipes/linkify-it-py/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 92fe76ad70ec6389e9b0445a3c6ab38f14f7e67859a4b61a765c09b62d284f91
```
Comment (chrisjsewell) [inline recipes/linkify-it-py/meta.yaml:10]:
```suggestion
  sha256: 90b632ee516bf523c007ee96aa14ffc7efe1ca4074a80b0df366d66922d6d087
```

File: recipes/linkify-it-py/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - uc-micro-py
```
Comment (chrisjsewell) [inline recipes/linkify-it-py/meta.yaml:23]:
```suggestion
    - uc-micro-py >=1.0.1,<2.0.0
```

**Comment by chrisjsewell** [comment]
@conda-forge/help-python this is ready for review thanks

**Review by dopplershift — APPROVED** [review]

**Pull request merged**